core-bundle/contao/themes/flexible/styles/main.css

```diff
@@ -1355,7 +1355,7 @@ table.tl_listing {
 	min-height: 16px;
 }
 
-.tl_content.draft > *, .tl_folder.draft > *, .tl_file.draft > *, .hover-row.draft > td {
+.tl_content.draft .tl_content_left, .tl_folder.draft .tl_left, .tl_file.draft .tl_left, .hover-row.draft > td {
```

Comment:
This does not work for regular content elements because they do not have a `.tl_content_left` class. `.tl_content.draft .cte_type, .tl_content.draft .cte_preview, .tl_content.draft .limit_toggler` would do the trick and make it look like this:

<img width="821" alt="" src="https://github.com/user-attachments/assets/9b095203-f148-46d0-bb95-9d193f7cb131" />

Not sure if that is easier to understand though.

Reply:
@leofeyer we could also do the following since we already know what classes we'll have in case you do not want to use the `:not selector` (would also work with nested elements / wrapped ones since they are not really a child but a following `list` element)

```css
.tl_content.draft :is(.cte_type, .cte_preview, .limit_toggler)
```

```css
.tl_content.draft .cte_type,
.tl_content.draft .cte_preview,
.tl_content.draft .limit_toggler
```

Reply:
I don't think that there is a big difference between `:is()` and `:not()` when it comes to performance. As far as I recall, `:has()` is more problematic here.

If we really want `. tl_content_right` to have a different opacity than the rest of the row, I would go for `:not(.tl_content_right)`, simply because then it does not matter which other classes there are.

Reply:
For the content elements only `.cte_preview` should have a lower opacity in my opinion. Nested elements can be disregarded as they won't render in the front end if the parent is disabled.

Reply:
@leofeyer @fritzmg I did implement the suggested not selector in https://github.com/contao/contao/pull/7951/commits/0cbc5f745fa50ec451beb19229fb96874fbc81e6

This also features the cte_type not having an opacity as fritz suggested:
![image](https://github.com/user-attachments/assets/8f9b60ea-27a5-4400-bae5-ecb6724157a3)